migration/migrating_4_1_4/configuring-replication-repository-4-1-4.adoc

```diff
@@ -1,5 +1,5 @@
 [id='configuring-replication-repository-4-1-4']
-= Configuring a replication repository
+= Configuring object storage
```

Comment:
This seems confusing. It sounds like you are configuring storage for general reasons. I understand the change, as the user isn't really configuring a replication repository.
How about something like: _Configuring object storage for the replication repository_